content/en/infrastructure/resource_catalog/_index.md

```diff
@@ -82,27 +82,27 @@ The Security tab provides a security-centric view of your infrastructure and all
 
 By viewing misconfigurations, signals, identity risks, and vulnerabilities associated with resources, you can address security concerns without needing to spend time and effort to gather additional security context.
 
-{{< img src="/infrastructure/resource_catalog/resource-catalog-doc-img-3.png" alt="The Resource Catalog page showing the Security tab, grouped by resource type" width="100%">}}
+{{< img src="/infrastructure/resource_catalog/resource-catalog_security_tab.png" alt="The Resource Catalog page showing the Security tab, grouped by resource type" width="100%">}}
 
 ### List view
 
-You can filter, sort, and group resources in the Resource Catalog list by cloud platform, resource type, account, team, region, tags, and security risks to scope down to the resources that matter to your current context. For example, you may find it helpful to filter by your team name, or to scope the misconfigurations to particular environments and clusters. 
+Under the Security tab, you can filter, sort, and group resources in the Resource Catalog list by cloud platform, resource type, account, team, region, tags, and security risks to scope down to the resources that matter to your current context. For example, you may find it helpful to filter by your team name, or to scope the misconfigurations to particular environments and clusters. 
 
 To filter the list to see a subset of resources you're most interested in, select resource categories or types from the resource selector on the left, or use the query cards and individual facets at the top of your list view. To group by any default or resource specific attributes as well as tags, use the **Group by** dropdown on the top right of your list view. You can additionally sort by different columns or attributes by clicking on the column names. If you are looking for a particular resource, you can search for the resource name directly in the search bar. 
 
-To access the relevant cloud console for any resource in your list, use the **Open Cloud Console** button to be redirected.
+{{< img src="/infrastructure/resource_catalog/resource-catalog_security_tab_list.png" alt="Resource Catalog Security tab sorting by the list view" >}}
 
-{{< img src="/infrastructure/resource_catalog/resource-catalog-doc-img-4.png" alt="Resources owned by the cloud-networks team grouped by resource type" width="100%">}}
+**Pro Tip**: If you use [Datadog Teams][4], select the **Teams** toggle on the left panel, then select the toggle for the teams to which you're assigned to view only resources assigned to those teams. In addition, export your Resource Catalog list as a CSV file from the top right corner of the list.
```

Comment:
```suggestion
**Note**: If you use [Datadog Teams][4], select the **Teams** toggle on the left panel, then select the toggle for the teams to which you're assigned to view only resources assigned to those teams. In addition, export your Resource Catalog list as a CSV file from the top right corner of the list.
```

Reply:
I don't believe we do "Pro Tips"

Reply:
good catch! I think that was there before but I didn't catch that :)